examples/dreambooth/train_dreambooth_lora_sdxl.py

```diff
@@ -496,7 +502,11 @@ def __getitem__(self, index):
             instance_image = instance_image.convert("RGB")
         example["instance_images"] = self.image_transforms(instance_image)
 
-        example["instance_prompt_ids"] = self.instance_prompt_hidden_states
+        if self.instance_prompt_hidden_states is not None:
+            example["instance_prompt_ids"] = self.instance_prompt_hidden_states
+        else:
+            example["instance_prompt_tokens_one"] = tokenize_prompt(self.tokenizer_one, self.instance_prompt)
+            example["instance_prompt_tokens_two"] = tokenize_prompt(self.tokenizer_two, self.instance_prompt)
```

Comment:
This will re-tokenize the same prompt every time we pull from the dataloader. I think we can just tokenize it ahead of time.